CMakeLists.txt

```diff
@@ -332,7 +332,9 @@ if (NOT HAVE_CXX_ATOMICS_WITHOUT_LIB)
     endif()
     set(LDFLAGS "${LDFLAGS} -latomic")
   else()
-    message(FATAL_ERROR "Host compiler appears to require libatomic, but cannot find it.")
+    message(STATUS "Host compiler appears to require libatomic, but cannot find it.")
+    message(STATUS "Enabling mutex lock workaround.")
```

Comment:
Mind removing the dots at the end? We don't use this style for all other STATUS messages.